src/V3SenTree.h

```diff
@@ -97,4 +97,221 @@ class SenTreeFinder final {
     }
 };
 
+//######################################################################
+// SenExprBuilder constructs the expressions used to compute if an
+// AstSenTree have triggered
+
+class SenExprBuilder final {
```

Comment:
Let's put this into `V3SenExprBuilder.h`

Reply:
Done.